src/validator.js

```diff
@@ -471,4 +474,21 @@ export default class Validator {
       return ARCH_DEFAULT;
     }
   }
+
+  _markEmptyFiles(addonMetadata) {
+    var emptyFiles = [];
+
+    return this.io.getFiles()
+      .then((files) => {
+        for (let filename in files) {
+          if (files[filename].uncompressedSize === 0) {
```

Comment:
I’d go for adding uncompressedSize as an alias; also means I should add an extra test for directories then. Thanks!